packages/elements/src/select/index.ts

```diff
@@ -357,6 +365,8 @@ export class Select extends ControlElement implements MultiValue {
   protected firstUpdated (changedProperties: PropertyValues): void {
     super.firstUpdated(changedProperties);
     this.addEventListener('keydown', this.onKeyDown); /* keydown when select is closed */
+
+    registerOverflowTooltip(this, () => this.labelRef.value ? this.labelRef.value.textContent || '' : '');
```

Comment:
The `TooltipCondition` is missing cause the tooltip does not work.

Reply:
Thanks, it is fixed